manifests/devconsole/0.1.0/devconsole-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -26,8 +26,32 @@ spec:
       name: gitsourceanalyses.devconsole.openshift.io
       displayName: Git Source Analysis
       description: Triggers analysis of a codebase defined as a GitSource.
-  description: The operator that enables the Developer perspective in OpenShift 4.x
-  displayName: OpenShift Developer Console
+  description: >
+    The operator that enables a developer-focused perspective in OpenShift 4. 
+    This enables a perspective (view) switcher to transition between the traditional 
+    Kubernetes and Cluster Administration console, referred to as Administrator, 
+    to this new perspective Developer perspective.
+
+    This new Developer perspective provides a high-level abstraction over
+    Kubernetes and OpenShift primitives to allow developers to focus on
+    their application development.
+    
+    Within the Developer perspective there are these main areas:
+
+    * Add - The place to create and build the application using one of this method:
+    
+      - Importing source code from Git
+      - Deploying an existing image
+      - Browse a catalog to deploy or connect application serves
```

Comment:
application services?

Reply:
I had copied this from the readme. `git blame @baijum` :P

Reply:
@Avni-Sharma Good catch. @sbose78 Please update README also. Thanks!

Reply:
Done!